support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -192,38 +198,53 @@ class DigitalPackEmailFields(
       gift_end_date = "gift end date placeholder", // TODO need to pull it through from when we create the sub
     ))
 
-  private def corpRedemption =
-    wrap("digipack-corporate-redemption", directOrCorpFields("Group subscription"))
-
-  private def directThankYou(paymentMethodWithSchedule: PaymentMethodWithSchedule) =
-    wrap("digipack", DirectDSAttributes(
-      directOrCorpFields(SubscriptionEmailFieldHelpers.describe(paymentMethodWithSchedule.paymentSchedule, billingPeriod, currency, promotion)),
-      country = user.billingAddress.country.name,
-      date_of_first_payment = formatDate(SubscriptionEmailFieldHelpers.firstPayment(paymentMethodWithSchedule.paymentSchedule).date),
-      trial_period = "14", //TODO: depends on Promo code or zuora config
-      paymentFields(paymentMethodWithSchedule.paymentMethod, directDebitMandateId)
-    ))
+  private def corpRedemption(subscriptionNumber: String) =
+    wrap("digipack-corporate-redemption", directOrCorpFields("Group subscription", subscriptionNumber))
+
+  private def directThankYou(directPurchase: DigitalSubscriptionDirectPurchaseCreated)(implicit ec: ExecutionContext) = {
+
+    val promotion = paperFieldsGenerator.getAppliedPromotion(
+      directPurchase.purchaseInfo.promoCode,
+      user.billingAddress.country,
+      ProductTypeRatePlans.digitalRatePlan(directPurchase.product, touchPointEnvironment).map(_.id).getOrElse("")
```

Comment:
this `""` was already there but it can probably be ironed out one day

Reply:
(previously was on line 123 of SendThankYouEmail.scala 
https://github.com/guardian/support-frontend/pull/2792/files#diff-9c81d24a8e7c52abfde88fe9288edc673a3fe9ddc9fb734ae7b31bac675a3a6eL123 )